scaletest/terraform/action/coder_helm_values.tftpl

```diff
@@ -34,7 +34,11 @@ coder:
     - name: "CODER_URL"
       value: "${access_url}"
     - name: "CODER_PROVISIONERD_TAGS"
-      value: "scope=organization"
+      value: "scope=organization,deployment=${deployment}"
+    - name: "CODER_PROVISIONER_DAEMON_NAME"
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.name
```

Comment:
Is this something we should add by default to the coder-provisioner Helm chart?

Reply:
I think it could be useful as a default as long as you can override it